src/storage/btree.c

```diff
@@ -14339,8 +14369,8 @@ btree_find_rightmost_leaf (THREAD_ENTRY * thread_p, BTID * btid, VPID * pg_vpid,
  * Note: Find the page identifier for the first/last leaf page of the B+tree index.
  */
 static PAGE_PTR
-btree_find_boundary_leaf (THREAD_ENTRY * thread_p, BTID * btid, VPID * pg_vpid, BTREE_STATS * stat_info,
-			  BTREE_BOUNDARY where)
+btree_find_boundary_leaf_with_repl_desync_check (THREAD_ENTRY * thread_p, BTID * btid, VPID * pg_vpid,
+						 BTREE_STATS * stat_info, BTREE_BOUNDARY where, bool * desync_occured)
```

Comment:
enough to pass by address; passing by pointer everywhere is c-style and Cubrid coding guidelines do not enforce using this c-style aspect
```suggestion
						 BTREE_STATS * stat_info, BTREE_BOUNDARY where, bool & desync_occured)
```
also, the value of this flag is never reset anywhere